src/pages/MyKiva/MyKivaPage.vue

```diff
@@ -101,11 +103,8 @@ export default {
 					inviterName: this.userInfo.userAccount?.inviterName ?? null,
 				};
 				this.loans = myKivaQueryResult.my?.loans?.values ?? [];
-				if (bpSidesheetLoan?.lend?.loan) {
-					const bpLoanId = bpSidesheetLoan.lend.loan.id;
-					const filteredLoans = this.loans.filter(loan => loan.id !== bpLoanId);
-					this.loans = [bpSidesheetLoan.lend.loan, ...filteredLoans];
-				}
+				this.sidesheetLoan = bpSidesheetLoan?.lend?.loan ?? {};
```

Comment:
The old code also ensured the loan appeared in the carousel. Can we still ensure that happens if the current user has lent to it?

Reply:
The link will be used in email showing users loans they funded, so we ideally want it in the borrower carousel. But we'd need to check to make sure the user actually lent towards it, like you mentioned.

Reply:
Yes eventually, but we need to change the updates carousel in general to use new filter.